Core/UIViewControllerExtension.swift

```diff
@@ -74,3 +74,21 @@ extension UIViewController {
         present(controller, animated: true, completion: nil)
     }
 }
+
+extension Core.Link: UIActivityItemSource {
+
+    public func activityViewControllerPlaceholderItem(_ activityViewController: UIActivityViewController) -> Any {
+        AppUrls().removeATBAndSource(fromUrl: url)
+    }
+
+    public func activityViewController(_ activityViewController: UIActivityViewController,
+                                       itemForActivityType activityType: UIActivity.ActivityType?) -> Any? {
+        AppUrls().removeATBAndSource(fromUrl: url)
+    }
+
+    public func activityViewController(_ activityViewController: UIActivityViewController,
+                                       subjectForActivityType activityType: UIActivity.ActivityType?) -> String {
+        title ?? ""
```

Comment:
There are other sensible defaults that could be used here. I went with an empty string since that's what we've been doing already, but this could default to the URL itself or some predefined string like "Link shared via DuckDuckGo".